sql/org_mozilla_firefox/migrated_clients_v1/view.sql

```diff
@@ -0,0 +1,65 @@
+CREATE OR REPLACE VIEW `moz-fx-data-shared-prod.org_mozilla_firefox.migrated_clients_v1` AS
```

Comment:
We should remove the _v1 from the view name unless there's a compelling reason to version it per https://github.com/mozilla/bigquery-etl#views